dev/create-release/release-util.sh

```diff
@@ -636,12 +637,22 @@ function kick_gpg_agent {
 # Do maven command to set version into local pom
 function maven_set_version { #input: <version_to_set>
   local this_version="$1"
-  echo "${MVN[@]}" versions:set -DnewVersion="$this_version"
-  "${MVN[@]}" versions:set -DnewVersion="$this_version" | grep -v "no value" # silence logs
-  #Hacking around nonstandard maven submodules
+  # Omid needs an hbase profile for tagging (doesn't matter which one, the shims need special handling anyway)
+  if [[ "$PROJECT" == "phoenix-omid" ]]; then
+    local variant="${OMID_VARIANTS[1]}"
+  fi
+  echo "${MVN[@]}" ${variant:+"$variant"} versions:set -DnewVersion="$this_version"
+  "${MVN[@]}" ${variant:+"$variant"} versions:set -DnewVersion="$this_version" | grep -v "no value" # silence logs
+  #Hacking around nonstandard maven submodules for phoenix only
   for i in phoenix-hbase-compat-*; do
     if [ -e "$i" ]; then
-     "${MVN[@]}" -pl $i versions:set -DnewVersion="$this_version" | grep -v "no value" # silence logs
+     "${MVN[@]}" ${variant:+"$variant"} -pl $i versions:set -DnewVersion="$this_version" | grep -v "no value" # silence logs
+    fi
+  done
+  #Omid has an even more nonstandard maven structure, and needs more hacks
+  for i in hbase-shims/hbase-*; do
+    if [ -e "$i" ]; then
+     sed -i -e "0,\#<version>.*</version>#{s##<version>${this_version}</version>#}" $i/pom.xml
```

Comment:
I tried to find the alternative with options in the version maven plugin to update multi-level parent-child relationship but it seems we have no option other than this hack

Reply:
Long term we should fix the build to build both set of artifacts in one pass, like connectors and tephra does.